modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -428,21 +433,23 @@ FeatureFloodCount::communicateAndMerge()
    * After each of those processors has merged that information, it'll be sent to the primary
    * processor where final consolidation will occur.
    */
-  if (_distribute_merge_work)
+  const auto n_merging_procs = numberOfDistributedMergeHelpers();
+
+  if (n_merging_procs > 1)
   {
     auto rank = processor_id();
-    bool is_merging_processor = rank < _n_vars;
+    bool is_merging_processor = rank < n_merging_procs;
 
     if (is_merging_processor)
       recv_buffers.reserve(_app.n_processors());
 
-    for (MooseIndex(_n_vars) i = 0; i < _n_vars; ++i)
+    for (MooseIndex(n_merging_procs) i = 0; i < n_merging_procs; ++i)
```

Comment:
Not ready to let go of `MooseIndex` and jump on the `make_range` bandwagon?

Reply:
More like I'm a manager and in deep voice, "Why does nobody tell me about these things?"

Reply:
Since this is ready to go, I'll leave it alone for now as I have another set of changes riding on this one. I can make a pass at dropping `MooseIndex` later.